pom.xml

```diff
@@ -49,7 +49,6 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-log4j2</artifactId>
-			<version>2.3.4.RELEASE</version>
```

Comment:
you deleted version for log4j2 as well, are you 100% sure it will work properly in Heroku?

Reply:
I made these changes because sonar marks it as a duplicated line in line 9 of pom sso, the version is already established